qcodes/instrument_drivers/basel/sp983c.py

```diff
@@ -38,6 +39,13 @@ def __init__(self, name: str, **kwargs: Any):
                            vals=Enum(30., 100., 300., 1e3, 3e3, 10e3, 30e3,
                                      100e3, 1e6))
 
+        self.add_parameter('bias_volt',
```

Comment:
shall we avoid shortenings for readability sake ? :)
```suggestion
        self.add_parameter('bias_voltage',
```